x-pack/plugins/observability_solution/observability_shared/public/locators/infra/asset_details_flyout_locator.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SerializableRecord } from '@kbn/utility-types';
+import rison from '@kbn/rison';
+import { LocatorDefinition, LocatorPublic } from '@kbn/share-plugin/common';
+
+export type AssetDetailsFlyoutLocator = LocatorPublic<AssetDetailsFlyoutLocatorParams>;
+
+export interface AssetDetailsFlyoutLocatorParams extends SerializableRecord {
```

Comment:
I am wondering if we want to support the hosts' page filters here as well 🤔 (I could be missing something here). Maybe we can have the same base of properties for the hosts locator and asset details flyout/view ( So have the `_a` base everywhere, the `tableProperties` in the hosts' page and asset details flyout, and the `assetDetails` in the asset details flyout and asset details view and share the parts of types between the locators, that way it will be easier to maintain IMO), wdyt?

Reply:
`_a` is not a param in the url for asset details, adding it will require other changes, out of the scope of this issue, we can create a follow-up issue for it

Reply:
Right, just checked, no need to add it there we don't have the filters like in hosts view there. It's fine I was looking for some common params but we don't have the same base params it seems.